Assets/Scripts/Player/Health.cs

```diff
@@ -0,0 +1,251 @@
+    private bool crit = false;
+
+    //structs for status bools
+    private struct limbStatus
+    {
+        public bool bruised;
+        public bool bleeding;
+        public bool crippled;
+        public bool burned;
+        public bool numb;
+        public bool blistered;
+    }
+
+    //structs for dmgType values
+    private struct HealthInfo
+    {
+        public float brute;
+        public float burn;
+        public float toxic;
+        public float suffocation;
+        public float maxLimbHP;
+    }
+
+    private limbStatus[] statusBools = new limbStatus[7];
+    private HealthInfo[] limbHealthInfo = new HealthInfo[7];
+
+    //enum for entity types
+    public enum Entities
+    {
+        Organic,
+        Onebi,
+        Connie
+    }
+
+    //enum for each body part
+    private enum SkeletalSections
+    {
+        torso,
+        head,
+        leftArm,
+        rightArm,
+        leftLeg,
+        rightLeg,
+        overall
+    }
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        //Intializes overall player health
+        currentHealth = maxHealth;
+        currentBloodVol = maxBloodVol;
+
+        //Initilize the HP of each respective limb
+        limbHealthInfo[(int)SkeletalSections.torso].maxLimbHP = limbHealthInfo[(int)SkeletalSections.head].maxLimbHP =
+        limbHealthInfo[(int)SkeletalSections.leftArm].maxLimbHP = limbHealthInfo[(int)SkeletalSections.rightArm].maxLimbHP =
+        limbHealthInfo[(int)SkeletalSections.leftLeg].maxLimbHP = limbHealthInfo[(int)SkeletalSections.rightLeg].maxLimbHP = maxHealth;
+
+        //Initilizes Thresholds dependent on Entity type
+        Thresholds.InitializeThresholds();
+    }
+
+    //Updates the damage value of corresponding damage to corresponding body part targetted
+    public void dmgUpdate(int target, int dmgType , float dmg)
+    {
+        if (dmgType == 0)
+        {
+            limbHealthInfo[target].brute += dmg;
+            limbHealthInfo[(int)SkeletalSections.overall].brute += dmg;
+        }
+        else if (dmgType == 1)
+        {
+            limbHealthInfo[target].burn += dmg;
+            limbHealthInfo[(int)SkeletalSections.overall].burn += dmg;
+        }
+        else if (dmgType == 2)
+            limbHealthInfo[(int)SkeletalSections.overall].toxic += dmg;
+        else
+            limbHealthInfo[(int)SkeletalSections.overall].suffocation += dmg;
+
+        dmgEval(target, dmgType);
+        dmgDebug();
+       }
+
+    public void dmgEval(int target, int dmgType)
+    {
+        //Determines the status of an affected limb based on the brute damage
+        if (dmgType == 0)
+        {
+            //Calculates if a limb is bruised
+            if (statusBools[target].bruised == false && Random.Range(Thresholds.thresholds[(int)Entities.Organic].minBruise, Thresholds.thresholds[(int)Entities.Organic].maxBruise)* limbHealthInfo[target].maxLimbHP < limbHealthInfo[target].brute)

```

Comment:
bruising statuses shouldn't be determined by how much damage is taken, rather the type of damage taken (piercing/blunt/slashing)

Reply:
if(!statusBools[target].bruised